### PR TITLE
if u have this config u can preload in building very much easier

### PR DESCRIPTION
**Pull request by pouriaTetsRunner**
I DID IT BECAUSE I FOUND OUT
the build of project takes so long and had many bugs such error cancelling for trubleshooting and no compatibility with Linux 7.2.5 and Ubuntu 5.0.2
and i want to contribute with ur team more thx

### CODE REVIEW

**Comment by pouria1370** [comment]
good Pouria it s very valuable but i thinku must focus more on network compatibilty. to more i want u see face to face
for more conversation

File: Forked/.babelrc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+{
```
Comment (pouria1370) [inline Forked/.babelrc:1]: here i think will be better you change httprewuewsrt

**Review by pouria1370 — COMMENTED** [review]
its good

**Review by pouria1370 — CHANGES_REQUESTED** [review]
change it

**Comment by pouriaTetsRunner** [comment]
shure thx

**Pull request closed without merging**